### ISSUE TITLE
django 2.1.7 compatibility

### ISSUE THREAD

**Issue by pincoin** [SmileyChris/easy-thumbnails#512]
Hello,

Today I tried to upgrade easy-thumbnail from 2.5 to 2.6, but I encountered the following errors:

```
easy-thumbnails 2.6 has requirement django<2,>=1.8, but you'll have django 2.1.7 which is incompatible.
easy-thumbnails 2.6 has requirement pillow<4, but you'll have pillow 5.1.0 which is incompatible.
```

Please, fix the compatibility issue.

Thank you for your support.

**Comment by jrief** [comment]
Looking at `setup.py`, easy-thumbnails 2.6 has a requirement of "pillow<4", but only if used with Python-2.7, which is incompatible with Django-2.1 anyway.

Try an installation without dependency check and look what happens.

**Comment by pincoin** [comment]
Hello,

I use python 3.5.2 (pyenv) and Django 2.1.7 on Ubuntu 16.04.

I forced to upgrade your package easy-thumbnails from 2.5 to 2.6 by explicit version of `requirements.txt`.

I also had to specify Pillow version 5.4.1.

However, I see the following warnings when I command `pip install -r requirements.txt`.

```
easy-thumbnails 2.6 has requirement django<2,>=1.8, but you'll have django 2.1.7 which is incompatible.
easy-thumbnails 2.6 has requirement pillow<4, but you'll have pillow 5.4.1 which is incompatible.
```

I think it's really weird because the pyenv on Mac doesn't show errors with the same `requirements.txt`.
